package.json

```diff
@@ -79,9 +83,9 @@
     "@types/jest": "29.5.11",
     "@types/js-yaml": "4.0.9",
     "@types/node": "18.17.19",
-    "@types/react": "17.0.66",
+    "@types/react": "18.2.46",
     "@types/react-copy-to-clipboard": "5.0.7",
-    "@types/react-dom": "18.2.8",
+    "@types/react-dom": "18.2.18",
```

Comment:
Do we have to bump the react types? Since we are still using v17, I'm concerned about the types not matching the actual JS code being executed and causing weird errors.

Reply:
Yes We Have To Bump It Up. It's Still Compatible With v17. Bumping The Type To v18 Would Require Alot Of Changes In The Code And Would Also Require Bumping Up Other Dependencies Like Electron.. e.t.c
Meanwhile, Did You Encounter Any Weird Errors While Testing/Running The App?

Reply:
I haven't run into any issues in my testing. This just makes me a bit nervous, but I guess it should be fine. I definitely don't think we should go down that rabbit hole of upgrading React & Electron. That is going to be a painful project which I hope to get to one day.